test/fuzz/transformation_replace_id_with_synonym_test.cpp

```diff
@@ -2026,11 +2026,10 @@ TEST(TransformationReplaceIdWithSynonymTest,
                    .IsApplicable(context.get(), transformation_context));
 }
 
-// TODO(https://github.com/KhronosGroup/SPIRV-Tools/issues/4345): Improve this
-//  test so that it covers more atomic operations, and enable the test once the
-//  issue is fixed.
```

Comment:
You should reinstate this comment, since we weren't able to enable the test after all.

Reply:
Done.